### PR TITLE
docs(graphql): Update mapping Graphql to Dgraph

### PR DESCRIPTION
**Pull request by bucanero**
Fixes DOC-337

### CODE REVIEW

File: content/graphql/dgraph/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ weight = 13
 
 If you have an existing Dgraph instance and want to also expose GraphQL, you need to add a GraphQL schema that maps to your Dgraph schema.  You don't need to expose your entire Dgraph schema as GraphQL, but do note that adding a GraphQL schema can alter the Dgraph schema.
 
-Dgraph also allows type and edge names that aren't valid in GraphQL, so, often, you'll need to expose valid GraphQL names. Dgraph admits special characters and even different languages (see [here](https://docs.dgraph.io/query-language/#predicate-name-rules)), while the GraphQL Spec requires that type and field (predicate) names are generated from `/[_A-Za-z][_0-9A-Za-z]*/`.
+Dgraph also allows type and edge names that aren't valid in GraphQL, so, often, you'll need to expose valid GraphQL names. Dgraph admits special characters and even different languages (see [Predicate name rules]({{< relref "query-language/schema.md#predicate-name-rules">}})), while the GraphQL Spec requires that type and field (predicate) names are generated from `/[_A-Za-z][_0-9A-Za-z]*/`.
```
Comment (aaroncarey) [inline content/graphql/dgraph/index.md:13]:
```suggestion
Dgraph's native DQL syntax allows type and edge names that aren't valid in GraphQL; so, you'll often need to expose valid GraphQL names. Dgraph permits special characters, including Unicode characters from a variety of  languages (see [Predicate name rules]({{< relref "query-language/schema.md#predicate-name-rules">}})). Conversely, the [GraphQL specification on naming](https://spec.graphql.org/June2018/#sec-Names) requires that entity names, including types and fields (predicate), are composed of ASCII characters and generated as follows: `/[_A-Za-z][_0-9A-Za-z]*/`.
```

File: content/graphql/dgraph/index.md [hunk 2]
```diff
@@ -93,7 +93,11 @@ type Movie {
 }
 ```
 
-*Note: the current behavior requires that when two fields are mapped to the same Dgraph predicate both should have the same `@search` directive.  This is likely to change in a future release where the underlying Dgraph indexes will be the union of the `@search` directives, while the generated GraphQL API will expose only the search given for the particular field.  Allowing, for example, dgraph predicate name to have `term` and `hash` indexes, but exposing only term search for GraphQL movies and hash search for GraphQL people.*
+{{% notice "note" %}}
+The current behavior requires that when two fields are mapped to the same Dgraph predicate both should have the same `@search` directive.
```
Comment (aaroncarey) [inline content/graphql/dgraph/index.md:97]:
```suggestion
In Dgraph's current GraphQL implementation, if two fields are mapped to the same Dgraph predicate, both should have the same `@search` directive.
```

File: content/graphql/dgraph/index.md [hunk 3]
```diff
@@ -93,7 +93,11 @@ type Movie {
...
+This is likely to change in a future release where the underlying Dgraph indexes will be the union of the `@search` directives, while the generated GraphQL API will expose only the search given for the particular field.
```
Comment (aaroncarey) [inline content/graphql/dgraph/index.md:98]: I'd cut these two lines (98 and 99), as we already hint that this behavior might change, and detailing possible future implementations isn't a best practice.

File: content/graphql/dgraph/index.md [hunk 4]
```diff
@@ -136,5 +140,3 @@ Future features are likely to include:
 * Generating a first pass GraphQL schema from an existing dgraph schema.
```
Comment (aaroncarey) [inline content/graphql/dgraph/index.md:140]: I know it isn't 100% within the scope of this PR, but I'd be in favor of cutting the whole "## Roadmap" section... But maybe it makes more sense for me to file a task to remove "roadmap" /  detailed forward-looking statements in general. Not a merge blocker, just a thought.

**Review by aaroncarey — APPROVED** [review]
Approved, but with a few suggested edits :)

File: content/graphql/dgraph/index.md (hunk 4 above)
Comment (bucanero) [reply content/graphql/dgraph/index.md]: I removed the `Roadmap` section

File: content/graphql/dgraph/index.md (hunk 3 above)
Comment (bucanero) [reply content/graphql/dgraph/index.md]: Lines removed.

**Pull request merged**